Gemfile

```diff
@@ -3,21 +3,15 @@ source 'http://rubygems.org'
 gemspec
 
 gem 'simplecov', '>= 0.4.0', :require => false, :group => :test
-
-mongoid_version =
-  case ENV['MONGOID_VERSION']
-  # TODO: uncomment this option when all specs pass for 6.0
-  # when /^6/
-    # '~> 6.0'
-  when /^5/
-    '~> 5.0'
-  when /^4/
-    '~> 4.0'
-  when /^3/
-    '~> 3.1'
-  else
-    # TODO: change default value to '~> 6.0' when all specs pass for 6.0
-    '~> 5.0'
-  end
-
-gem 'mongoid', mongoid_version
+case version = ENV['MONGOID_VERSION'] || '~> 6.0.0'                                                             
+when /6/
+  gem 'mongoid', '~> 6.0.0'                                                                                     
+when /5/                                                                                                      
+  gem 'mongoid', '~> 5.0'                                                                                     
+when /4/                                                                                                      
+  gem 'mongoid', '~> 4.0'                                                                                     
+when /3/                                                                                                      
+  gem 'mongoid', '~> 3.1'                                                                                     
+else                                                                                                          
+  gem 'mongoid', version                                                                                      
+end
```

Comment:
Use Gemfile from master and just uncomment lines 10-11, and change line 20 to `'~> 6.0'`